src/test/java/com/webonise/rbs/RbsApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package com.webonise.rbs;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class RbsApplicationTests {
```

Comment:
Remove s from the class name 'RbsApplicationTests'. Always consider class as noun so it should not be pluralised.